cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2895,3 +2897,56 @@ def test_organism_cell_type_ontology_term_id__visium_in_tissue_0__na(self, valid
         validator.validate_adata()
         # Passes visium check but fails organism_cell_type_ontology_term_id check
         assert validator.errors == [error_message]
+
+    def test_organism_tissue_type_ontology_term_id(self, validator_with_fruitfly_adata):
+        validator = validator_with_fruitfly_adata
+        obs = validator.adata.obs
+        obs.loc[obs.index[0], "organism_tissue_ontology_term_id"] = "FBbt:00007337"  # valid descendant of FBbt:10000000
+        validator.validate_adata()
+        assert not validator.errors
+
+    @pytest.mark.parametrize(
+        "organism_tissue_ontology_term_id,error",
+        [
+            (
+                "UBERON:0000001",  # Wrong ontology
+                "ERROR: 'UBERON:0000001' in 'organism_tissue_ontology_term_id' is not a valid ontology term id "
+                "of 'FBbt'.",
+            ),
+            (
+                "FBbt:10000000",  # Must be descendant of FBbt:10000000, not itself
+                "ERROR: 'FBbt:10000000' in 'organism_tissue_ontology_term_id' is not an allowed term id.",
+            ),
+            (
+                "FBbt:00007002",  # FBbt:00007002 is an explicitly forbidden term
+                "ERROR: 'FBbt:00007002' in 'organism_tissue_ontology_term_id' is not allowed.",
+            ),
+            (
+                "FBbt:00007294",  # FBbt:00007002 descendant, an explicitly forbidden ancestor
+                "ERROR: 'FBbt:00007294' in 'organism_tissue_ontology_term_id' is not allowed. Descendant terms of "
+                "'FBbt:00007002' are not allowed.",
+            ),
+            (
+                "na",  # Allowed for other organisms, not allowed if organism is zebrafih
+                "ERROR: 'na' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
+            ),
+            (
+                "unkown",
+                "ERROR: 'unkown' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
```

Comment:
```suggestion
                "unknown",
                "ERROR: 'unknown' in 'organism_tissue_ontology_term_id' is not a valid ontology term id of 'FBbt'.",
```

Reply:
can you correct this for zebrafish too lol

Reply:
yeah i gotchu haha